File: Server/Schedule/Schedule.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Schedule
{
    public class Schedule
    {
        public Schedule(string lesson, string teacher, int room, string description, DateTime time)
        {
            Lesson = lesson;
            Teacher = teacher;
            Room = room;
            Description = description;
            Time = time;
        }
        public string Lesson { get; set; }
        public string Teacher { get; set; }
        public string Description { get; set; }
        public int Room { get; set; }
        public DateTime Time { get; set; }
    }
}

File: app/src/main/java/server_connection/Schedule.java
package server_connection;

import android.os.Build;
import androidx.annotation.RequiresApi;
import java.time.LocalDateTime;
import java.time.format.DateTimeFormatter;

@RequiresApi(api = Build.VERSION_CODES.O)
public class Schedule
{
    public String Lesson;
    public String Teacher;
    public String Description;
    public String Room;
    public String Time;

    public Schedule(String lesson, String teacher, String room, String description, String time)
    {
        this.Lesson = lesson;
        this.Teacher = teacher;
        this.Room = room;
        this.Description = description;
        this.Time = time;
    }

    public LocalDateTime GetLDC() {
        return LocalDateTime.parse(this.Time);
    }

    public String GetLDC(DateTimeFormatter format) {
        return LocalDateTime.parse(this.Time).format(format);
    }
}